tests/tasks/test_employee_salary_history.py

```diff
@@ -11,18 +11,6 @@ def test_extract_dw_employee_salary_history_load_sql_query(
     assert task_extract_dw_employee_salary_history_complete.sql_query is not None
 
 
-def test_task_extract_qb_employee_appointments_complete_has_required_fields(
```

Comment:
After realizing that test is exercising a shared task, made sense to move it into a new, dedicated test file `tests/tasks/test_shared.py`.

Reply:
Since this is an uncommon scenario, some comments or additions to `README.md` might be helpful to provide context for the `test_shared.py` module

Reply:
Good plan!